src/Microsoft.Azure.NotificationHubs/RegistrationDescription.cs

```diff
@@ -217,6 +215,15 @@ internal string FormattedETag
 
         internal abstract RegistrationDescription Clone();
 
+        /// <summary>
+        /// Returns platform-specific Psn handle.
+        /// </summary>
+        /// 
+        /// <returns>
+        /// platform-specific Psn handle.
+        /// </returns>
+        public string PsnHandle => GetPnsHandle();
```

Comment:
The property name should be `PnsHandle`.

Reply:
Changed